falcon/api.py

```diff
@@ -52,66 +52,27 @@ def process_request(self, req, resp):
                                 the on_* responder
                         \"""
 
-                    def process_response(self, req, resp)
-                        \"""Post-processing of the response (after routing).
+                    def process_resource(self, req, resp, resource):
+                        \"""Process the request after routing.
+
+                        Args:
+                            req: Request object that will eventually be
+                                routed to an on_* responder method
+                            resp: Response object that will be routed to
```

Comment:
Similar comment to above. Suggested text:

"resp: Response object that will be passed to the responder"